src/js/components/Pagination/__tests__/Pagination-test.tsx

```diff
@@ -530,8 +532,7 @@ describe('Pagination', () => {
           options={[10, 20, 50, 100]}
           value={itemPerPage} // Set value of Select to itemPerPage
           onChange={({ option }) => {
-            // Update itemPerPage state when Select value changes
-            setItemPerPage(option);
+            if (typeof option === 'number') setItemPerPage(option);
```

Comment:
also why are we checking the type of option

Reply:
I added that because OnChange will accept an argument of type OptionType, and step is a type of number, if we don't check the type, it will show an error.

Reply:
I don't think we need to adjust this test -- it looks like the test has all options defined as numbers, so I don't think an error will be thrown. Let's keep the PR lean if possible.